app/Models/Foundation/Summit/Events/Presentations/Actions/PresentationActionType.php

```diff
@@ -46,6 +47,22 @@ class PresentationActionType extends SilverstripeBaseModel
      */
     private $order;
 
+    /**
+     * @ORM\OneToMany(targetEntity="models\summit\AllowedPresentationActionType", mappedBy="type", cascade={"persist"}, orphanRemoval=true, fetch="EXTRA_LAZY")
+     * @var AllowedPresentationActionType[]
+     */
+    private $selection_plans;
```

Comment:
collection should be called something more meaningful like assigned_selection_plans

Reply:
done